lib/tasks/deploy.rake

```diff
@@ -48,7 +48,14 @@ namespace :deploy do
     sh "docker build -t #{tag} ."
     sh "docker push #{tag}"
 
-    trigger_deployment(application, build, branch) if branch == 'staging'
+    next unless branch == 'staging'
+
+    deployed_apps = [application]
```

Comment:
It should have been `apps_to_deploy`, which doesn't sound good. I'm always having trouble naming things "to-be-done". `deploy_apps` sounds like a function, but it's ok.